src/main/kotlin/BUILD.release.bazel

```diff
@@ -43,6 +43,10 @@ java_binary(
         "@com_github_jetbrains_kotlin//:lib/jvm-abi-gen.jar",
         "@com_github_jetbrains_kotlin//:lib/kotlin-compiler.jar",
     ],
+    jvm_flags = [
+        "-XX:-MaxFDLimit",
+        "-XX:+IgnoreUnrecognizedVMOptions",
```

Comment:
why is this here?

Reply:
Initially thought the flag was not supported in Java 8 so added this but it seems Java 8 is supported so this is not needed. https://github.com/bazelbuild/rules_kotlin/pull/848 alone is fine I think.